CHANGELOG.md

```diff
@@ -55,7 +55,23 @@ All notable changes for each version of this project will be documented in this
     - The `IgxBottomNav` component exposes `disableAnimations` property which determines whether the contents should animate when switching the selected item. The property is set to `true` by default which means that the animations are disabled. 
 - `IgxOverlayService`
     - `detach` and `detachAll` methods are added to `IgxOverlayService`. Calling `detach` will remove all the elements generated for the related overlay, as well as clean up all related resources. Calling `detachAll` will remove all elements generated by any call to `IgxOverlay` `attach`, and will clean up all related resources. _Note: calling `hide` or `hideAll` will not clean generated by the service elements and will not clean up related resources._
- 
+- `IgxCombo`
+    - Any changes to `IComboItemAdditionEvent.addedItem` will be reflected in the item added to the data.
+    - `IgxComboComponent.onAddition` is now cancelable. You can prevent the item from being added by setting the event args `cancel` property to `false`.
```

Comment:
to prevent event you should set `cancel` to `true` IMO

Reply:
Ooops, thanks! 🍻